Sources/Commands/SwiftTool.swift

```diff
@@ -122,13 +122,13 @@ public class SwiftTool<Options: ToolOptions> {
         // Bind the common options.
         binder.bindArray(
             parser.add(
-                option: "-Xcc", kind: [String].self,
+                option: "-Xcc", kind: [String].self, strategy: .one,
                 usage: "Pass flag through to all C compiler invocations"),
             parser.add(
-                option: "-Xswiftc", kind: [String].self,
+                option: "-Xswiftc", kind: [String].self, strategy: .one,
                 usage: "Pass flag through to all Swift compiler invocations"),
             parser.add(
-                option: "-Xlinker", kind: [String].self,
+                option: "-Xlinker", kind: [String].self, strategy: .one,
```

Comment:
`nargs` feels really wrong to me for a Swift API. Plus, it's not really the same behaviour as `nargs`. The `one` case for example is the way I chose to tell the option to parse a multiple-value argument one-by-one, but through appending can end up being more than one values.

Any other ideas?

* `parser.add(option: "-Xlinker", kind: [String].self, strategy: .oneByOne)`
* `parser.add(option: "-Xlinker", kind: [String].self, parse: .oneByOne)`